apps/src/templates/studioHomepages/TeacherAssignablesCatalog.jsx

```diff
@@ -1,52 +1,78 @@
+  },
+
   render() {
     const { codeOrgUrlPrefix, isRtl } = this.props;
 
     return (
-      <ContentContainer heading={i18n.teacherCoursesHeading()} isRtl={false}>
-        <ResourceCard
-          title={i18n.teacherCourseHoc()}
-          description={i18n.teacherCourseHocDescription()}
-          image="hourofcode"
-          buttonText={i18n.learnMore()}
-          link={`${codeOrgUrlPrefix}/hourofcode/overview`}
+      <div>
+        <ContentContainer
+          heading={i18n.teacherCoursesHeading()}
+          description={i18n.teacherCoursesDescription()}
           isRtl={isRtl}
-        />
-        <ResourceCard
-          title={i18n.teacherCourseElementary()}
-          description={i18n.teacherCourseElementaryDescription()}
-          image="elementary"
-          buttonText={i18n.learnMore()}
-          link={`${codeOrgUrlPrefix}/educate/curriculum/elementary-school`}
-          isRtl={isRtl}
-        />
-        <ResourceCard
-          title={i18n.teacherCourseMiddle()}
-          description={i18n.teacherCourseMiddleDescription()}
-          image="middleschool"
-          buttonText={i18n.learnMore()}
-          link={`${codeOrgUrlPrefix}/educate/curriculum/middle-school`}
-          isRtl={isRtl}
-        />
-        <ResourceCard
-          title={i18n.teacherCourseHighOlder()}
-          description={i18n.teacherCourseHighDescription()}
-          image="highschool"
-          buttonText={i18n.learnMore()}
-          link={`${codeOrgUrlPrefix}/educate/curriculum/high-school`}
+        >
+          <ResourceCard
+            isJumbo={true}
+            title={i18n.teacherCourseElementary()}
+            description={i18n.teacherCourseElementaryDescription()}
+            buttonText={i18n.learnMore()}
+            link={`${codeOrgUrlPrefix}/educate/curriculum/elementary-school`}
+            isRtl={isRtl}
+          />
+          <div style={styles.horizontalSpacer}/>
+          <ResourceCard
+            isJumbo={true}
+            title={i18n.teacherCourseMiddle()}
+            description={i18n.teacherCourseMiddleDescription()}
+            buttonText={i18n.learnMore()}
+            link={`${codeOrgUrlPrefix}/educate/curriculum/middle-school`}
+            isRtl={isRtl}
+          />
+          <div style={styles.horizontalSpacer}/>
+          <ResourceCard
+            isJumbo={true}
+            title={i18n.teacherCourseHighOlder()}
+            description={i18n.teacherCourseHighDescription()}
+            buttonText={i18n.learnMore()}
+            link={`${codeOrgUrlPrefix}/educate/curriculum/high-school`}
+            isRtl={isRtl}
+          />
+        </ContentContainer>
+
+        <ContentContainer
+          heading={i18n.teacherCourseHoc()}
+          description={i18n.teacherCourseHocDescription()}
           isRtl={isRtl}
-        />
-      </ContentContainer>
+          linkText={i18n.teacherCourseHocLinkText()}
+          link={`${codeOrgUrlPrefix}/hourofcode/overview`}
```

Comment:
I'm not sure I'm reading this right but wanted to confirm teachers go to code.org/hourofcode/overview but students go to code.org/learn?